iml-agent/Cargo.toml

```diff
@@ -21,12 +21,12 @@ spinners = "1.0.0"
 libc = "0.2.58"
 log = "0.4.6"
 prettytable-rs = "0.8"
-reqwest = { version = "0.9.17", features = ["default-tls", "tls"] }
+reqwest = { version = "0.9.18", features = ["default-tls", "tls"] }
```

Comment:
This conflicts with #998.  Other patches sets version to "0.9"

Reply:
I'll change to 0.9